docs/development/getting-started.rst

```diff
@@ -6,22 +6,22 @@ Development dependencies
 
 Working on ``cryptography`` requires the installation of a small number of
 development dependencies in addition to the dependencies for
-:doc:`/installation`. These are handled by the use of ``tox``, which can be
+:doc:`/installation`. These are handled by the use of ``nox``, which can be
 installed with ``pip``.
 
 .. code-block:: console
 
     $ # Create a virtualenv and activate it
     $ # Set up your cryptography build environment
-    $ pip install tox
+    $ pip install nox
     $ # Specify your Python version here.
-    $ tox -e py310
+    $ nox -e tests -p py310
 
 OpenSSL on macOS
 ~~~~~~~~~~~~~~~~
 
 You must have installed `OpenSSL`_ via `Homebrew`_ or `MacPorts`_ and must set
-``CFLAGS`` and ``LDFLAGS`` environment variables before running ``tox``
+``CFLAGS`` and ``LDFLAGS`` environment variables before running ``nox``
```

Comment:
Looks like this is a section we need to update for the new env var?

Reply:
Yes, but can we do that in a seperate PR, since it's not part of the cahnges her?